### DISCUSSION TITLE
How to convert XSD file to Java POJO

### DISCUSSION THREAD

**Discussion by durgeshrjkumar** [pdvrieze/xmlutil#150] [Q&A]
How can we convert our xsd or xml file to pojo classes annotated with support of this library in java?

**Comment by pdvrieze** [comment]
Unfortunately I haven't completed xsd support at this point (it is a quite complex standard, and its semantic model is not as easy as the document model). As such you will have to do mapping by hand. Additionally, this is a Kotlin library relying on the kotlin serialization plugin for creating the serializers. As such you cannot directly create objects in Java, but the objects created can be used from Java. Note that getting access to serializers from Java is also not as convenient as it is from Kotlin.

**Comment by durgeshrjkumar** [comment]
I was looking for command like 
C:\Users\User\Downloads\uAPI_WSDLschema\uapi\air_v1_0>xjc Air1.xsd
This command generates pojo. 
Thanks for timely reply. :)

**Comment by pdvrieze** [comment]
I figured you were looking for an equivalent to that (generating Kotlin classes). Unfortunately this doesn't exist at this point (it is a long-term work in progress project).
